assets/src/js/Components/ArchivePicker/ArchivePicker.js

```diff
@@ -233,7 +265,50 @@ export default function ArchivePicker() {
     }
   }, [loading, loaded, images, error, searchText, pageNumber, dispatch]);
 
-  const includeInWp = async (ids = []) => {
+  const currentBlock = wp.data.select('core/block-editor').getSelectedBlock();
+
+  const getImageDets = useCallback(async id => {
```

Comment:
I'd prefer to rename to `getImageDetails` instead which is more understandable !